### ISSUE TITLE
Support running `apply` and other commands with input from stdin

### ISSUE THREAD

**Issue by mattrobenolt** [segmentio/topicctl#28]
I'm working towards building something that leverages `topicctl` for our kafka cluster in kubernetes, and with this, my plan is to have a simple wrapper around topicctl that listened to a Kubernetes ConfigMap containing cluster configs. On ConfigMap changes, it'd apply to the cluster.

To better support that, it'd be nice to be able to pass everything through stdin. I looked briefly at code thinking it'd be simple, but there are some assumptions made with file system layout (discovering cluster config.yaml) and not handling multiple documents (#26).

I think ideally, it'd be nice to pass one big document through stdin that contained the cluster config as well as the topics through stdin. Then our loop in Kubernetes will be:

```
while:
  configmap <- configmap.wait_for_change()
  configmap -> subprocess topicctl apply -
```

Am I just off base on wanting to use `topicctl` in this manner? It seems like a pretty obvious thing for us to do. We can work around the issues by having another intermediate step where we write all the configs to disk first, then call `topicctl apply topics/*yml` or whatever, but is an unnecessary step if we can pass through stdin.

Let me know your thoughts, we haven't started building anything for this yet, just brainstorming.

**Comment by yolken-segment** [comment]
Thanks for the suggestion. This is not unreasonable given that `kubectl` and other, similar type tools typically allow input from stdin. However, it will require a bit of non-trivial refactoring in the various entrypoints, so I would advise sticking with the "write to disk" intermediate step for now. From past experience, the latter is not an uncommon pattern when trying to get tools with non-Kubernetes-optimized interfaces working in Kubernetes.

**Comment by mattrobenolt** [comment]
Yeah, for sure. And I didn't want to start a large refactor to support this without at least some communication. It's easy enough to write to disk as an intermediate solution.

**Comment by yolken-segment** [comment]
Cool! https://github.com/segmentio/topicctl/issues/26 is a lot easier to implement, so I'll start with that one.

**Comment by mattrobenolt** [comment]
Just to add another bump for support here, another use case for us would be another tool which generated the configs that we could write to stdout and pass in to topicctl. 

Something like: `sentry config generate-topicctl | topicctl apply -`

I think the doors will open up for lots of possibilities if we can do this for us.

**Comment by vishnu-anil** [comment]
Hi is my understanding correct here that topicctl cannot be run as a kubernetes pod? I tried to run this with cmd and args as below for the pod container but fails with below error

`time="2022-07-31 21:37:28" level=error msg="unknown command "apply /opt/topics/*yaml --skip-confirm" for "topicctl""`

```
command: ["topicctl"]
args: ["apply", "/opt/topics/*yaml", "--skip-confirm" ]
```
Is this a limitation? Please confirm
